Documentation/flexvolume.md

```diff
@@ -35,6 +41,12 @@ To configure Flexvolume in Rancher, specify this Kubelet flag as shown below:
 
 Restart Kubelet in order for this change to take effect.
 
+In Kubernetes 1.9.x, you must provide this directory when deploying the [rook-operator](/cluster/examples/kubernetes/rook-operator.yaml) by setting the environment variable`FLEXVOLUME_DIR_PATH`. For example:
```

Comment:
The sections for Rancher and CoreOS look the same to me. What if we create a more general section for all the duplicated info and then if there are specific differences, they can have their own section?

Reply:
i thought about it. But i thought this was the easiest way to convey this information. Even if its duplicate. Users tends to search for a topic, for example "coreos" and just read everything until the next heading. Do you have a suggestion to make it more readable?